tests/std/tests/P0595R2_is_constant_evaluated/test.cpp

```diff
@@ -23,6 +23,7 @@ int cube(const int x) {
 
 #ifdef __clang__
 #pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunknown-warning-option" // TRANSITION, Clang 10

```

Comment:
Because this is TRANSITION, I'll overlook the fact that the warning isn't sorted 😺

Reply:
Sorting would ruin the entire point: This is here to suppress the warning clang 9 generates when it sees the warning suppression on the next line for `-Wconstant-evaluated` which it does not recognize.

Reply:
...and now that I have a `// TRANSITION, clang 10` I'm thinking I should have made the changes to the feature-test macro test conditional and checked them in as well. 🤷‍♂